tests/playwright/src/model/pages/cli-tools-page.ts

```diff
@@ -89,42 +89,10 @@ export class CLIToolsPage extends SettingsPage {
     });
   }
 
-  public async installTool(toolName: string, version: string = '', timeout = 60_000): Promise<this> {
+  public async installTool(toolName: string, timeout = 60_000): Promise<this> {
```

Comment:
We are breaking contract here. In next iteration we will have option to install particular version back. I would be for keeping api intact and throw an error when version is used now in this function call.

Reply:
You need to check if it was not used elsewhere already, minikube has usage that wont be affected, but other extensions might be.